src/hooks/useVaultTypes.ts

```diff
@@ -1,11 +1,33 @@
 import { useMemo } from "react"
+import { Address } from "wagmi"
 
 import { VaultType } from "@/lib/types"
+import { useConcentratorVaultTypeByAsset } from "@/hooks/useConcentratorVaultTypeByAsset"
 
-export function useIsCurveCompounder(type: VaultType) {
+export function useIsCurveVault(type: VaultType) {
   return useMemo(() => type === "curve", [type])
 }
 
-export function useIsTokenCompounder(type: VaultType) {
+export function useIsTokenVault(type: VaultType) {
   return useMemo(() => type === "token", [type])
 }
+
+export function useIsConcentratorCurveVault(asset: Address) {
+  const vaultTypeAvailable = useConcentratorVaultTypeByAsset()
```

Comment:
I'm confused by this variable name. What do you mean "available"?

Reply:
One of the big issues in the concentrator now is, it doesn't mention the vault type of the target asset or primary asset. The concentrator APY (the reward and the compounder vault) needs to know the vault type for the fallback methods. Token and curve might be assumed to be the vault types for this first fcGLP/tricrypto concentrator, however, the moment a new concentrator is introduced, the fallback methods should be updated to reflect the new target asset or primary asset. I hardcoded at the moment in useConcentratorVaultTypeByAsset this vaultType for this arbi fork chain.

Reply:
Available here means, hardcoded

Reply:
Got it, thanks.